bluepysnap/circuit_validation.py

```diff
@@ -619,6 +623,14 @@ def validate(config_file, skip_slow, only_errors=False, print_errors=True):
     if "networks" in config:
         errors += validate_networks(config, skip_slow)
 
+    if _check_partial_circuit_config(config):
```

Comment:
I was thinking of converting FATAL errors into WARNINGS for partial configs, but it might be unnecessarily complex